app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -63,6 +65,14 @@ def in_person_enrollment?(user)
         ProofingComponent.find_by(user: user)&.document_check == Idp::Constants::Vendors::USPS
       end
 
+      def device_profiling_failed?(user)
```

Comment:
To me it feels like a confusing double negative to say that the user passes the test if device_profiling_failed is false.  It could be named `device_profiling_passed?` and return true when the user passes. Or, I would also find the name `blocked_by_device_profiling?` clearer if you still want it to return false on a pass.